common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -257,4 +257,7 @@ public Properties cryptoConf() {
     return CryptoUtils.toCryptoConf("spark.network.crypto.config.", conf.getAll());
   }
 
+  public long maxChunksBeingTransferred() {
+    return conf.getLong("spark.network.shuffle.maxChunksBeingTransferred", Long.MAX_VALUE);
```

Comment:
I feel there are too many "by default disabled" optimizations, which can't benefit most of the spark users. I'm ok to leave it disabled for now, but we should definitely revisit it and think about reasonable default values. also cc @tgravescs @zsxwing 

BTW, please document this config in `configuration.md`